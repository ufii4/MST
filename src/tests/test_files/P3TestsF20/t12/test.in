n 8
i 1;6;1.5
i 1;0;2
clear
edge_count
i 1;2;3.4
i 3;4;1.5
i 6;5;2.3
i 8;7;1.1
i 0;1;1.1
i 0;2;2.1
i 0;3;1.6
i 0;4;4.5
i 0;5;1.9
i 0;6;3.4
i 0;7;3.1
i 8;0;0.7
i 1;3;1.1
i 1;4;2.1
i 1;4;3.1
i 1;6;2.2
i 5;1;3.2
i 6;1;2.1
i 7;1;2.2
i 8;1;1.2
i 1;8;0.1
mst
d 1;8
edge_count
exit
