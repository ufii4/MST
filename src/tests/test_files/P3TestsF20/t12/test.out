success
success
success
success
edge count is 0
success
success
success
invalid argument
success
success
success
success
success
success
success
invalid argument
success
success
success
success
success
success
success
invalid argument
invalid argument
mst 12.00
invalid argument
edge count is 15
